README.md

```diff
@@ -30,27 +30,35 @@ Install [Visual Studio version 15.9-preview1 or later](https://visualstudio.micr
 
 If you are using `dotnet` or `nuget`, you can use the Azure Artifact Credential Provider by adding it to [NuGet's plugin search path](https://github.com/NuGet/Home/wiki/NuGet-Cross-Plat-Credential-Plugin#plugin-installation-and-discovery). This section contains both manual and scripted instructions for doing so.
 
-### Manual installation: Windows
+### Installation on Windows
+
+#### Automatic PowerShell script
+
+[PowerShell helper script](helpers/installcredprovider.ps1)
+- To install netcore, run `installcredprovider.ps1`
+- To install both netfx and netcore, run `installcredprovider.ps1 -AddNetfx`. The netfx version is needed for nuget.exe.
+
+#### Manual installation on Windows
 
 1. Download the latest release of [Microsoft.NuGet.CredentialProvider.zip](https://github.com/Microsoft/artifacts-credprovider/releases)
 2. Unzip the file
-3. Copy the `netcore` (and optionally `netfx` for nuget.exe) directory from the extracted archive to `$env:UserProfile\.nuget\plugins`
+3. Copy the `netcore` (and `netfx` for nuget.exe) directory from the extracted archive to `$env:UserProfile\.nuget\plugins`
 
-### Automatic PowerShell installation:
+Using the above is recommended, but as per [NuGet's plugin discovery rules](https://github.com/NuGet/Home/wiki/NuGet-Cross-Plat-Credential-Plugin#plugin-installation-and-discovery), alternatively you can install the credential provider to a location you prefer, and then set the environment variable NUGET_PLUGIN_PATHS to the .exe of the credential provider found in plugins\netfx\CredentialProvider.Microsoft\CredentialProvider.Microsoft.exe. For example, $env:NUGET_PLUGIN_PATHS="my-alternative-location\CredentialProvider.Microsoft.exe". 
```

Comment:
This will break `dotnet` (at least until https://github.com/NuGet/Home/issues/8151 is fixed). Please update this guidance to make that clear.

Reply:
Thanks! I'll remove it from under dotnet and leave it for under nuget

Reply:
Hmm, nuget has the issue as well? Maybe I'll just add a link to the bug here. I'll think about this one.

Reply:
The "Linux and Mac" instructions below are perfectly fine, since dotnet is the only possibility there.

You don't need to remove it from here IMO, it should just indicate that setting the envvar to the .exe credprovider will break dotnet on windows, and setting the envvar to the .dll will break nuget.exe and MSBuild on windows.

Reply:
Therefore customers that use both dotnet and nuget.exe/MSBuild (.NET Framework) should probably avoid that envvar for now.

Reply:
I added "Note that if you are using both nuget and dotnet, this environment variable is not recommended due to this issue: https://github.com/NuGet/Home/issues/8151"

Reply:
Looks good for the windows section, but is unnecessary for the linux/mac section (running nuget.exe and the .exe credprovider is not supported)

Reply:
Thanks. I guess I'm ready for the weekend.